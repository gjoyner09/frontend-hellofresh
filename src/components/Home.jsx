export const Home = () => {
    return (
        <>
            <h1>Home</h1>
            <p>Welcome to my Hello Fresh App! Click the links above to see the weekly menus and recipes. Bon Apetit!</p>
        </>
    )
}