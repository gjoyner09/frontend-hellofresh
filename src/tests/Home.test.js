import React from 'react'
import renderer from 'react-test-renderer'
import Home from '../components/Home'

test('Homepage renders correctly', () => {
    const component = renderer.create(
        <Home></Home>
    )
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})