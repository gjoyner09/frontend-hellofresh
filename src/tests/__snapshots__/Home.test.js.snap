// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
Array [
  <h1>
    Home
  </h1>,
  <p>
    Welcome to MealHub! Click the links above to see the weekly menus and recipes. Bon Apetit!
  </p>,
]
`;

exports[`Homepage renders correctly 1`] = `
Array [
  <h1>
    Home
  </h1>,
  <p>
    Welcome to MealHub! Click the links above to see the weekly menus and recipes. Bon Apetit!
  </p>,
]
`;
